client/js/libs/handlebars/parse.js

```diff
@@ -78,6 +81,8 @@ module.exports = function parse(text) {
 		} else if (textPart.channel) {
 			const escapedChannel = Handlebars.Utils.escapeExpression(textPart.channel);
 			return `<span class="inline-channel" role="button" tabindex="0" data-chan="${escapedChannel}">${fragments}</span>`;
+		} else if (textPart.emoji) {
+			return `<span class="inline-emoji">${fragments}</span>`;
```

Comment:
I hope that someday we'll be able to display the reverse-lookup'ed text for an emoji in a tooltip. Great when an emoji doesn't show up (for compatibility or whatever reason), also good for accessibility, and things as simple as not recognizing what a damn emoji is meant to look like 😂

Reply:
I just paste in Google but yeah, that's one of the reasons lol.